core/ngsocket.cpp

```diff
@@ -745,7 +747,14 @@ BaseToPeerMessage NgSocket::parseBaseToPeerMessage(string_view data)
     return msg;
   }
 
-  msg.kind = BaseToPeerMessageKind::_from_integral(data[0]);
+  //TODO: refactor, dead code?
+  auto msgKind = BaseToPeerMessageKind::_from_integral_nothrow(data[0]);
+  if (msgKind) {
+    msg.kind = msgKind.value();
+  } else {
+    LOG_ERROR("invalid message kind: %d", data[0]);
```

Comment:
Change this to DEBUG. As base servers are sending some extra messages that clients do not yet/already understand it seems to be noisy for users.